pom.xml

```diff
@@ -3,7 +3,7 @@
 	<parent>
 		<groupId>org.jenkins-ci.plugins</groupId>
 		<artifactId>plugin</artifactId>
-		<version>1.532.3</version>
+		<version>1.626</version>
```

Comment:
I would recommend bumping to 1.642.x LTS baseline then, it's a good practice to test changes against Weekly releases. Update to Parent POM 2.x would be also helpfult, but it's likely out of the scope.

Reply:
I'll change to 1.642.x